### PR TITLE
slight improvements to the tournament bracket view

### PR DESCRIPTION
**Pull request by joram**

### CODE REVIEW

**Comment by brandonb927** [comment]
@joram does this PR replace https://github.com/battlesnakeio/play/pull/74?

File: play/apps/tournament/views/tournament_bracket.py [hunk 1]
```diff
@@ -61,8 +61,42 @@ def show_current_game(request, id):
...
     tournament_bracket = TournamentBracket.objects.get(id=id)
+
+    def num_snakes_and_games(total_snakes):
+        import math
+        game_count = math.ceil(total_snakes/8)
+        snakes_per_game = total_snakes/game_count
+        min_snakes_per_game = math.floor(snakes_per_game)
+        max_snakes_per_game = math.ceil(snakes_per_game)
+        snakes_advancing = game_count*2
+        return game_count, (min_snakes_per_game, max_snakes_per_game), snakes_advancing
```
Comment (dlsteuer) [inline play/apps/tournament/views/tournament_bracket.py]: I think this return is pretty horrendous, and going to be a maintenance burden.

Comment (brandonb927) [reply play/apps/tournament/views/tournament_bracket.py]: I agree, this is hard to parse here and where it's used.

File: play/apps/tournament/views/tournament_bracket.py [hunk 2]
```diff
@@ -61,8 +61,42 @@ def show_current_game(request, id):
...
+    progression_details = []
+    round = 1
+    total_snakes = len(tournament_bracket.snakes)
+    while total_snakes > 8:
+        games, (min_s, max_s), advancing = num_snakes_and_games(total_snakes)
+        print(round, games, (min_s, max_s), advancing)
+        progression_details.append({
+            "round": round,
+            "num_games": games,
+            "snakes_per_game": "{}-{}".format(min_s, max_s) if min_s!=max_s else "{}".format(min_s),
```
[the remark this answers is not in the document]
Comment (joram) [reply play/apps/tournament/views/tournament_bracket.py]: and here I was just getting used to `.format` instead of `"..." % (...)` :p

File: play/apps/tournament/views/tournament_bracket.py (hunk 1 above)
Comment (joram) [reply play/apps/tournament/views/tournament_bracket.py]: > @joram does this PR replace #74?

it does not, I closed the other one and will redoing after the refactor

**Comment by dlsteuer** [comment]
@joram i made a couple fixes to this and merged in master on #98 so i'm going to close this one. I like the start of the stats on the tournament bracket page, this will help with what we talked about
